src/PublicApiGeneratorTests/Event_visibility.cs

```diff
@@ -0,0 +1,130 @@
+﻿using PublicApiGeneratorTests.Examples;
+using System;
+using Xunit;
+
+namespace PublicApiGeneratorTests
+{
+    public class Event_visibility :  ApiGeneratorTestsBase
```

Comment:
This file is a direct copy of `Property_visibility` with a few search and replaces and with the tests of differing accessor visibility removed. (Event accessor visibility can differ in .NET assemblies but not in C#.)